qemu/tests/cfg/blockdev_stream_to_invalid_node.cfg

```diff
@@ -25,8 +25,11 @@
     rebase_mode = unsafe
     mount_point = "/var/tmp"
     qemu_force_use_drive_expression = no
+    required_qemu_version = [6.1.0, )
 
     virtio_scsi:
-        qmp_error_msg = "Conflicts with use by ${device_tag} as 'root', which uses 'write' on %s"
+        qmp_error_msg_6_0 = "Conflicts with use by ${device_tag} as 'root', which uses 'write' on %s"
+        qmp_error_msg_6_1 = "permissions 'write' are both required by block device '${device_tag}' (uses node '%s' as 'root' child)"
```

Comment:
what about qmp_error_before_6_1 qmp_error_after_6_1, for err msg is changed since qemu 6.1